daemon/network.go

```diff
@@ -222,6 +222,8 @@ func (daemon *Daemon) releaseIngress(id string) {
 		return
 	}
 
+	daemon.deleteLoadBalancerSandbox(n)
```

Comment:
Very fair question and one discussed a bit in https://github.com/docker/libnetwork/issues/2097.   Ultimately, this function was copied into libnetwork about 6 months ago but the original never got removed.   As it was, this was dead code.  There is a function of the same name and behavior in libnetwork and it is not exported.   It would be possible to just export said function (changing the libnetwork API, but in a small way).  However, given that 
    1. I'm working on a patch in the near term to refactor of load balancing to improve scalability
    2. This refactoring might affect how we expose said function
    3. We are trying to get this regression fixed for the immanent release
It seemed more prudent to make the smaller change to fix the regression and then clean up the duplicate code in the larger refactorization.

Reply:
Thanks; yes it's not new code, so I'm good with keeping it for a follow-up. Just was looking at it, and it felt like a _lot_ of error conditions were ignored, so left that comment for later investigation if needed. 👍